### PR TITLE
fix(@clayui.com): Fix broken links

### PR DESCRIPTION
**Pull request by kresimir-coko**
This PR fixes the broken links as described in [2837](https://github.com/liferay/clay/issues/2837).

In the [attached Google sheet](https://docs.google.com/spreadsheets/d/1upQV01S9YEOjXXeFAnEIZmnpoTncBLalZcfsMQ7WecU/edit#gid=0) I've marked ones that were straightforward with green (eg. `links.html` to `link.html`), questionable ones with yellow, and the last (metalJS) is unfixed here because it is, I believe, the issue of the metalJS site not clayui.com

### CODE REVIEW

File: clayui.com/content/docs/components/css-management-toolbar.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ description: 'Management toolbar is an extension of Toolbar. It is a combination
 </div>
 
 <div class="clay-site-alert alert alert-warning">
-    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/dropdown.html#alignment">Dropdown Alignment</a>.
+    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/drop-down.html#alignment">Dropdown Alignment</a>.
```
Comment (bryceosterhaus) [inline clayui.com/content/docs/components/css-management-toolbar.md]: We should point to `clayui.com` and not localhost

File: clayui.com/content/docs/components/navigation/css-application-bar.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ description: 'This component is used as the main navigation of Liferay DXP.'
 <div class="clay-site-alert alert alert-info">Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/satellite-components/navigation/">Application Bar Pattern</a> for a more in-depth look at the motivations and proper usage of this component.</div>
 
 <div class="clay-site-alert alert alert-warning">
-    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/dropdown.html#alignment">Dropdown Alignment</a>.
+    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/drop-down.html#alignment">Dropdown Alignment</a>.
```
Comment (bryceosterhaus) [inline clayui.com/content/docs/components/navigation/css-application-bar.md]: lets remove localhost

File: clayui.com/content/docs/components/navigation/css-navigation-bar.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ description: 'A navigation bar, is a horizontal bar that provides several access
 </div>
 
 <div class="clay-site-alert alert alert-warning">
-    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/dropdown.html#alignment">Dropdown Alignment</a>.
+    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/components/drop-down.html#alignment">Dropdown Alignment</a>.
```
Comment (bryceosterhaus) [inline clayui.com/content/docs/components/navigation/css-navigation-bar.md]: localhost

**Comment by kresimir-coko** [comment]
Hey @bryceosterhaus I've removed the links to localhost you caught, good eye.
There are some more links that are just like `#1` but the anchor text is `localhost...`, here's an example [css-sidebar.md#L154](https://github.com/liferay/clay/blob/master/clayui.com/content/docs/components/css-sidebar.md#L154). So I've left those in for now seeing as they don't point to anything bad, just the text they're displaying is techy and I don't know what they should be referencing

**Pull request merged**